lib/batch/institutional_nucats_grants_repo_child_collections.rb

```diff
@@ -0,0 +1,82 @@
+# https://github.com/galterlibrary/digital-repository/issues/678
+# New collections under NUCATS Grants Repository
+#   - Federal Non-NIH Awards
+#   - U-series
+#   - R-Series
+#     - move R01, R03, R21 collections here
+#   - Grants Resources
+#     - Pink Sheets
+#     - Biosketches
+
+# NUCATS Grants Repository
+parent_nucats_grants = Collection.find("f2bf6e1d-0e32-4ce2-a52e-bb0522d5708d")
+
+# Federal Non-NIH Awards
+fed_non_nih = Collection.new(
+  title: "Federal Non-NIH Awards",
+  tag: ["grant proposals", "research support", "peer-reviewed grants"]
+)
+fed_non_nih.apply_depositor_metadata(parent_nucats_grants.depositor)
+fed_non_nih.save!
+
+parent_nucats_grants.members << fed_non_nih
+parent_nucats_grants.save!
+
+# U-Series
+u_series = Collection.new(
+  title: "U-Series",
+  tag: ["grant proposals", "research support", "peer-reviewed grants"]
+)
+u_series.apply_depositor_metadata(parent_nucats_grants.depositor)
+u_series.save!
+
+parent_nucats_grants.members << u_series 
+parent_nucats_grants.save!
+
+# R-Series
+r_series = Collection.new(
+  title: "R-Series",
+  tag: ["grant proposals", "research support", "peer-reviewed grants"]
+)
+r_series.apply_depositor_metadata(parent_nucats_grants.depositor)
+r_series.save!
+
+parent_nucats_grants.members << r_series 
+parent_nucats_grants.save!
+
+# Add R01, R03, R21 collections
+# note: removing these collections from NUCATS Grants Repository collection 
+# can be done on the frontend
+r01 = Collection.find("6a9690f5-664a-4654-a5ca-c1c1ca99db9b")
+r03 = Collection.find("49945521-3b35-4efd-9e77-e909900f1604")
+r21 = Collection.find("af85d02b-4b6c-48db-8cac-b82f4075e1f0")
+r_series.memebers << r01
```

Comment:
`memebers` ! 

![slowpoke_meme](https://user-images.githubusercontent.com/1932651/62803353-cd6c9800-baaf-11e9-9bb1-f49319e522ef.jpg)
(see urban dictionary's defintion of a **memeber** at your own risk)

Reply:
haha, nice catch.